client/my-sites/stats/wordads/highlight-card-simple.jsx

```diff
@@ -0,0 +1,14 @@
+import { Card } from '@automattic/components';
+
+function HighlightCardSimple( { heading, icon, value } ) {
```

Comment:
For another variant of a highlight card, could you place it along with others [here](https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/highlight-cards)? This way, we have all cards centralised making it easier if we decide to unify them or combine them somehow. We could also add a storybook story for it.

Reply:
My thinking was that I'd open up a maintenance task to refactor the various highlight cards and then move them into the component. We also have a variant here:

https://github.com/Automattic/wp-calypso/tree/trunk/client/my-sites/stats/stats-email-top-row

The new HighlightCardSimple component and the TopCard component are mostly logic free so they should factor together well. Any objection to addressing this in a follow-up PR?

Reply:
Sounds good. It will be great to refactor the cards as a maintenance task.

Reply:
Opened a task for this [here](https://github.com/Automattic/wp-calypso/issues/72493).